scripts/upload.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+process.env.NODE_ENV = 'production';
+
+const AWS = require('aws-sdk');
+const path = require('path');
+const fs = require('fs');
+
+let s3 = new AWS.S3({apiVersion: '2006-03-01'});
+let env = process.env.FLM_ENV
+let uploadParams = {Bucket: '', Key: '', Body: ''};
+let project_name = require(path.join(process.cwd(), 'package.json')).name;
+let base_dir = './build/assets';
+console.log(process.cwd(), project_name);
+
+// We could turn on uploading for everyone with this.
+// if (!env) {
+//   env = "local";
+//   console.log("FLM_ENV is empty, setting to 'local'.");
+// }
+
+switch (env) {
+  case "development":
+    uploadParams.Bucket = "assets.dev.flmcloud.net";
```

Comment:
Could we make the bucket names configured as a variable? Since this project is not FLM-specific